prody/apps/prody_apps/prody_pca.py

```diff
@@ -99,9 +99,16 @@ def prody_pca(coords, **kwargs):
                 pca.calcModes(nmodes)
                 ensemble = dcd
         else:
-            pca.buildCovariance(dcd, aligned=kwargs.get('aligned'))
-            pca.calcModes(nmodes)
-            ensemble = dcd
+            if len(dcd) > 1000:
+                pca.buildCovariance(dcd, aligned=kwargs.get('aligned'))
+                pca.calcModes(nmodes)
+                ensemble = dcd
+            else:
+                ensemble = dcd[:]
+                if not kwargs.get('aligned'):
+                    ensemble.iterpose(quiet=True)
+                pca.performSVD(ensemble)
+            nmodes = pca.numModes()
```

Comment:
Indeed, the nmd file has lots of modes in it with this line here instead of the number the user requested. The slicing isn't applied to scipion and npz files anyhow so they contain lots of modes regardless, but that's ok. The main use of the app besides the new one for scipion is for NMWiz and then these files aren't written anyway.